NuGet.config

```diff
@@ -20,7 +20,17 @@
     <add key="dotnet7-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7-transport/nuget/v3/index.json" />
     <!-- Used for the Rich Navigation indexing task -->
     <add key="richnav" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
   </packageSources>
+  <packageSourceMapping>
+    <packageSource key="dotnet-public"><package pattern="*" /></packageSource>
+    <packageSource key="dotnet-tools"><package pattern="*" /></packageSource>
+    <packageSource key="dotnet-eng"><package pattern="*" /></packageSource>
+    <packageSource key="dotnet7"><package pattern="*" /></packageSource>
+    <packageSource key="dotnet7-transport"><package pattern="*" /></packageSource>
+    <packageSource key="richnav"><package pattern="*" /></packageSource>
+    <packageSource key="nuget.org"><package pattern="Kerberos.NET" /></packageSource>
+  </packageSourceMapping>
```

Comment:
@ViktorHofer Who would be able to help me setup a reasonable way to consume Kerberos.NET in the test projects?

Reply:
(It is already part of dotnet org)

Reply:
Is kerberos.NET just a collection of nuget packages? In which feed are those available? Are you asking about how to consume the packages in test projects (which should be straightforward) or how to hook Kerberos.NET to dependency flow so that dotnet/runtime always consumes the very latest.

Reply:
It is one package, no extra dependencies. I would be fine with fixed version to begin with.

Reply:
That's due to the hacked NuGet config and not much of a surprise. If we can get Kerberos.NET published to one of the official feeds (eg. the transport one) this should be easy to solve.

Reply:
Sure, on it

Reply:
Should be available now -> https://dev.azure.com/dnceng/public/_artifacts/feed/dotnet-public/NuGet/Kerberos.NET/overview/4.5.174

Reply:
Great! Thanks a lot.